packaging/ios/build_and_publish.sh

```diff
@@ -37,10 +39,20 @@ function reset {
 }
 trap reset EXIT
 
+if [ -n "$kbfs_commit" ]; then
+  cd "$kbfs_dir"
+  echo "Checking out $kbfs_commit on client (will reset to $kbfs_branch)"
```

Comment:
the message should be `Checking out $kbfs_commit on kbfs` here too

Reply:
updated